core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.html

```diff
@@ -32,11 +32,21 @@ <h2 class="page-title">Workflows</h2>
       >
         <i nz-icon nzTheme="outline" nzType="file-add"></i>
       </button>
+      <button
```

Comment:
It's better if the download button only appears after a user has selected any workflow. You can easily achieve this with `ngIf`.

Reply:
Done.

Reply:
I use disabled button when no workflow checked, which make it more consistent with other button in the workflow section. ngIf could made the download button very abrupt.